aws/resource_aws_spot_fleet_request.go

```diff
@@ -256,6 +256,30 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 							Computed: true,
 							ForceNew: true,
 						},
+						"tag": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"key": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+
+									"value": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+								},
+							},
+							Set: spotFleetTagToHash,
+						},
+
+						"tags": {
```

Comment:
How does this attribute work, and how does it differ from `tag` set above?  For instance, in the `aws_instance` resource, you see this for configuration:

```
tags {
  Name = "HelloWorld"
}
```

Can you elaborate on how this `tags` attribute is used?